cmd/license-header-checker/main.go

```diff
@@ -77,7 +95,18 @@ func processFiles(options config.Options) {
 			wg.Add(1)
 			go func() {
 				defer wg.Done()
-				processFile(path, license, options)
+				op := processFile(path, license, options)
+
+				switch op {
+				case Skipped:
+					skipped++
```

Comment:
you must sync access to atomic values from goroutines, check: [https://golang.org/pkg/sync/atomic](https://golang.org/pkg/sync/atomic)

Reply:
👍